src/effects/RealtimeEffectManager.cpp

```diff
@@ -56,51 +56,6 @@ bool RealtimeEffectManager::RealtimeIsSuspended() const noexcept
    return mSuspended;
```

Comment:
Better, let's remove the function because it is used nowhere.

The function as written would not be reliable, because the caller would have no guarantee that the condition it reports remains the same, unless holding the mutax lock.  But then the function itself would deadlock, unless `std::recursive_mutex` were used instead.

It reminds me of: https://en.cppreference.com/w/cpp/memory/weak_ptr/expired

"This function is inherently racy if the managed object is shared among threads. In particular, a false result may become stale before it can be used. A true result is reliable."

Yet in our case we can't even say that one of the results remains reliable.